src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -498,13 +498,8 @@ static inline int MPIDI_OFI_do_put(const void *origin_addr,
     goto fn_exit;
   null_op_exit:
     mpi_errno = MPI_SUCCESS;
-    if (sigreq) {
-        *sigreq = MPIR_Request_create(MPIR_REQUEST_KIND__RMA);
-        MPIR_ERR_CHKANDSTMT((*sigreq) == NULL, mpi_errno, MPIX_ERR_NOREQ, goto fn_fail,
-                            "**nomemreq");
-        MPIR_Request_add_ref(*sigreq);
-        MPIDI_CH4U_request_complete(*sigreq);
-    }
+    if (sigreq)
+        *sigreq = MPIR_Request_create_complete(MPIR_REQUEST_KIND__RMA);
     goto fn_exit;
 }
```

Comment:
We can also change it in `do_get` and `do_accumulate` routines. They create completed-request for `rget|raccumulate|rget_accumulate` with similar code.

Reply:
Done.